### ISSUE TITLE
Backward compatibility only for minor version

### ISSUE THREAD

**Issue by juztas** [fast-data-transfer/fdt#23]
It should be backwards compatible for minor versions, but not for major. 0.26 cant work with 0.25 or lower, but 0.26 can work with 0.26.1/0.26.2

**Comment by fixy** [comment]
Yes, as seen today earlier FDT's does not understand new messages, need to think how can we handle this situation. Maybe on session start we can check versions and fallback into older FDT mode.

**Comment by hontvari** [comment]
Yyour time may be better spent than implementing a fallback, an error message about the incompatibility is enough. Compatibility were more important if there were an official debian package in the debian repository, but AFAIK there is not.
